sympy/utilities/tests/test_lambdify.py

```diff
@@ -581,3 +582,9 @@ def test_issue_2790():
 def test_ITE():
     assert lambdify((x, y, z), ITE(x, y, z))(True, 5, 3) == 5
     assert lambdify((x, y, z), ITE(x, y, z))(False, 5, 3) == 3
+
+def test_10375():
+    import numpy as np
+    m = Min(x, y, z)
+    l = lambdify([x,y,z], m)
```

Comment:
My main problem right now is : How to pass `numpy.min([args])` as `expr/Method/Function`
 So that I can pass the parameters `x, y, z` .`numpy.min`  wants `number` only; not the symbols `x,y,z`.I tried many ways but the same problem occurs. It look very simple problem.@asmeurer @moorepants @jksuom